README.md

```diff
@@ -1,24 +1,24 @@
-# README
+# Contact.me
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Keep track of your contacts, all in one place, anytime, anywhere!
 
-Things you may want to cover:
+## Technologies
```

Comment:
What versions of the technologies listed below?

Reply:
Done